Theme added and login input fixes

SakshumGusain opened the pull request:
- fixed mobile input field by only allowing [0-9] digits

- updates feedback page as per theme
![image](https://github.com/SamagraX-Stencil/ui-templates/assets/107306820/2efdb375-ce6c-477a-b2eb-ece14121a6dd)
![image](https://github.com/SamagraX-Stencil/ui-templates/assets/107306820/01a1eaa4-c204-4b4a-97cf-6e2154b49a0f)

Commit:
feedback theme and login input field updated

Commit:
minor changes

Commit:
fixes in feedback molecule

Commit:
share-button molecule

Commit:
minor changes

geeky-abhishek commented on src/molecules/share-buttons/assets/share.tsx:
```diff
@@ -0,0 +1,11 @@
+// @ts-ignore
+import React from 'react';
+const ShareIcon = (props: any) => {
+  return (
```
remove this svg and use materil ui icon

Commit:
resolved conversation

SakshumGusain committed:
conflict fixes

SakshumGusain commented:
- fixes conflicts
- Added theme to Chat UI molecule
- updated config of chat ui
- updated chat ui so that it use share-button molecule
![image](https://github.com/SamagraX-Stencil/ui-templates/assets/107306820/e276ff37-935a-4081-b0bb-d3d6540e7632)
![image](https://github.com/SamagraX-Stencil/ui-templates/assets/107306820/307b42cd-e6ce-42ad-9cfa-42492e4190d5)

prtkjakhar committed:
Merge branch 'dev' into theme-login
